connectors/src/connectors/microsoft/temporal/spreadsheets.ts

```diff
@@ -119,7 +119,8 @@ async function processSheet(
 
   const loggerArgs = {
     sheet: {
-      id: worksheet.id,
+      documentId: spreadsheetId,
```

Comment:
just checking the code, here it seems in `processSheet` we have in this order
- spreadsheet item
- worksheet internal id
- worksheet item
- spreadsheet internal id

=> really hard to read imho

Not in scope of this PR but I would refactor & rename: processSheet should get an object argument, in which, in that order & name:
- spreadsheet
- spreadsheetInternalId
- worksheet
- worksheetInternalId

Happy to take care of it

Reply:
All yours. It's your code :)

Reply:
Just here as eng runner making sure things run